mdb/mdb_launch.py

```diff
@@ -112,8 +112,10 @@ def launch(
     """
 
     logging.basicConfig(
-        filename="mdb-launch.log", encoding="utf-8", level=logging.DEBUG
+        encoding="utf-8",
```

Comment:
is there a reason this one is now printing to `stdout`? vs the other logs which are written to file? 

Is this better practice for the server end?

Reply:
That was an oversight on my part, and I didn't actually mean to commit without the filename. I found it easier to debug when it was just printing to `stdout`, as it meant one fewer tmux pane to `tail -f` the file, but either is fine. 

What I usually do with servers is log to `stdout` and have anything but warning or higher disabled unless some environment variable is set (e.g. `MDB_ENABLE_LOG`). The user probably doesn't need the detailed logs, and creating new log files may add confusion for a user. On the flip side, it means if something goes awry, the user always has the logs available to send back to you.

Maybe a compromise would be to save the log to the `~/.mdb` directory with a UUID? Then it doesn't pollute the file system but still lets you work out what went wrong when something goes wrong.

Reply:
good idea. I like it. For now I will keep as is because we are still in debug/build phase. When it's more feature stable I would be inclined to opt for option "save the log to the ~/.mdb directory with a UUID" :+1: